test-automation/config.properties

```diff
@@ -0,0 +1,6 @@
+digitalFormsBaseUrl = https://digitalforms-api-totdho-dev.pathfinder.gov.bc.ca/digitalforms/
+apiUsername = user
+apiPassword = RnJpIEp1b
+dbUsername = RSDFRM
+dbPassword = aresdeeEf7ar
+environment = dev
```

Comment:
These should not be committed to the repo. Could you remove these and add some environment variables?